src/server/models/list.ts

```diff
@@ -3,48 +3,34 @@ import { compact, toLower, trim } from "lodash";
 import { logger } from "server/services/logger";
 import { isGovUKEmailAddress } from "server/utils/validation";
 import { prisma } from "./db/prisma-client";
-import {
-  List,
-  CountryName,
-  ServiceType,
-  ListCreateInput,
-  ListUpdateInput,
-} from "./types";
-
-export async function findUserLists(
-  email: string
-): Promise<List[] | undefined> {
+import { List, CountryName, ServiceType, ListCreateInput, ListUpdateInput } from "./types";
+
+export async function findUserLists(email: string): Promise<List[] | undefined> {
```

Comment:
in my PR, I've disabled explicit returns. 

The Prisma type system is quite complex, but very useful. If you return a Prisma query, the inferred type will be the same as what you put into the Prisma query. For example, if you decide to ` select: { history: true } `, the inferred type will have a history property.

Reply:
is disabling explicit returns a configuration option?  can you pls provide a link to your PR that shows it?

Reply:
https://github.com/UKForeignOffice/lists/pull/497/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5

.eslintrc.js, under `rules`

Reply:
I think this is a minor point (correct me if wrong) but maybe something we can apply in future changes.  Reckon we can just get these changes done as they are.

Reply:
This was more of a comment, not a request.